server/migration/1617052908638-NewAccessRequestFlow.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  MigrationInterface,
+  QueryRunner,
+} from 'typeorm';
+
+export class NewAccessRequestFlow1617052908638 implements MigrationInterface {
+  name = 'NewAccessRequestFlow1617052908638';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "access_request" ADD "what_you_do" character varying array NOT NULL`);
+    await queryRunner.query(`ALTER TABLE "access_request" ADD "sponsor_name" character varying NOT NULL`);
+    await queryRunner.query(`ALTER TABLE "access_request" ADD "sponsor_email" character varying NOT NULL`);
+    await queryRunner.query(`ALTER TABLE "access_request" ADD "sponsor_phone" character varying NOT NULL`);
+    await queryRunner.query(`ALTER TABLE "access_request" ADD "justification" character varying NOT NULL`);
+    await queryRunner.query(`COMMENT ON COLUMN "user_notification_setting"."last_notified_date" IS NULL`);
+    await queryRunner.query(`ALTER TABLE "user_notification_setting" ALTER COLUMN "last_notified_date" SET DEFAULT null`);
```

Comment:
What are these `user_notification_setting` migrations meant to do?  Does it fix the issue with it always wanting to migrate that column?